users/tamashia/index.md

```diff
@@ -8,93 +8,7 @@ Aaron Tamashiro
-  * Performed data analysis using C/C++/ROOT programming.
-  * Evaluation of fission product yields from experimental data.
-  * Used oscilloscope for data acquisition and handled detectors that operated at high voltages and used a cryogenic fill system.
-  * Built the circuit board and soldered together electronics used for a cryogenic fill system.
-  * Assisted with experiments held at a cyclotron facility. 
-
-* **Undergraduate Research Intern at Lawrence Livermore National Laboratory:** June 2015 – September 2015
-  * Performed data analysis using C/C++/ROOT programming.
-  * Developed a Rutherford backscattering experiment for nondestructive material analysis using a vacuum chamber system.
-  * Used oscilloscope for data acquisition and handled detectors that operated at high voltages and used a cryogenic fill system.
-  * Built the circuit board and soldered together electronics used for a cryogenic fill system.
-  * Assisted with experiments held at a cyclotron facility. 
-
-***
-
-### Publications
-1. R.O. Hughes, J.T. Burke, R.J. Casperson, S. Ota, 1, S. Fisher, J. Parker, C.W. Beausang, M. Dag, P. Humby, J. Koglin, E. McCleskey, A.B. McIntosh, A. Saastamoinen, A.S. Tamashiro, E. Wilson, T.C. Wu, "The hyperion particle-$$\gamma$$ detector array", *Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment*, **856**, 47-52 (2017).
-
-***
-
-### Past Projects
-* **Evaluation of U-238 Fission Product Yields:** June 2017 - August 2017
-  * Used C/C++/ROOT programming to analyze gamma ray peaks to evaluate fission product yields of specific isotopes.
-
-* **Evaluation of U-235 Fission Product Yields:** June 2016 - September 2016
-  * Used C/C++/ROOT programming to analyze gamma ray peaks to evaluate fission product yields of specific isotopes.
-
-* **Rutherford Backscattering Chamber:** June 2015 - September 2015
-  * Set up vacuum chamber to use a silicon detector for alpha backscattering resulting in data acquisition for analysis of the material.
-
-***
-
-### Affiliations
-* **OSU Rifle Team:** September 2013 - Present
-  * Vice President, Member
-  
-* **OSU EPIC Movement:** Septemeber 2012 - Present
-  * Leader, Outreach
-
-* **American Nuclear Society:** September 2012 - Present
-  * Member
-
-***
-
-### Licenses
-* **DOE Certificate of Core Radiological Training:** June 2015 - Present
-
-***
-
-### Skills
-* **Programing Languages:** MATLAB, Python, C/C++/ROOT, and Latex
-* **Tools and Utilities:** MCNP, Google Sketch up, Solid works, Microsoft Word, Microsoft Excel, Microsoft Powerpoint, and Adobe Photoshop
-* **Skills**: Circuits, Vacuum systems, Oscilloscope, High Voltage, and Liquid Nitrogen.
-
-***
-
-### Personal interests
-Cars, lifting, and reading.
+Ben Pharn graduated from Oregon State University in 2019 with a Master's in Nuclear Science and Engineering.
+He is currently still at Oregon State University working under Dr. Camille Palmer.
```

Comment:
1. Flip lines 12 and 14? Helps with continuity.
2. Slight edit for clarity:
He is currently ~still~ pursuing a Ph.D. at Oregon State University working under Dr. Camille Palmer.

Reply:
Thanks Tony! I'll get those updated and pushed through!